sam.local.yml

```diff
@@ -20,4 +20,5 @@ Resources:
           S3_FEED_KEY: 'feed.xml'
           LOG_LEVEL: debug
           TZ: 'America/New_York'
+          RANDOMIZATION_METHOD: 'uniform'
```

Comment:
Probably document some methods for randomization here? I still am not sure how many variations we have here.